### PR TITLE
Fix overflow menu visibility for table

### LINKED ISSUE #86: Table Overflow menu should stay active while hovering the opened menu
## Detailed description

> Describe in detail the issue you're having.

The overflow menu icon disappears in the [Table storybook](https://angular.carbondesignsystem.com/?knob-size=md&knob-striped=true&knob-showSelectionColumn=true&selectedKind=Table&selectedStory=with%20overflow%20menu&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs) as soon as the mouse moves to select an option.

For comparison, (Carbon's Data Table)[http://www.carbondesignsystem.com/components/data-table/code] keeps the overflow icon visible while browsing the menu.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

Table

### PR DESCRIPTION
**Pull request by youda97**
closes #86

### CODE REVIEW

File: src/dialog/overflow-menu/overflow-menu.directive.ts [hunk 1]
```diff
@@ -51,6 +51,25 @@ export class OverflowMenuDirective extends DialogDirective {
...
+	}
+
+	toggle() {
+		super.toggle();
+		this.updateOpacity();
+	}
+
+	updateOpacity() {
+		if (this.elementRef.nativeElement.getAttribute("aria-expanded") === "true") {
+			this.elementRef.nativeElement.style.opacity = 1;
```
Comment (asudoh) [inline src/dialog/overflow-menu/overflow-menu.directive.ts]: Weighing in - Wondering if team wants CSS with attribute selector?

Comment (youda97) [reply src/dialog/overflow-menu/overflow-menu.directive.ts]: Well this should be something that needs to be set in the `"bx--overflow-menu--open"` class, which is currently not doing anything

Comment (cal-smith) [reply src/dialog/overflow-menu/overflow-menu.directive.ts]: So I'm not sure why this needs to be applied in the first place? Is the overflow menu getting `opacity: 0` in the table? if so why?

**Comment by youda97** [comment]
@cal-smith So currently it does not work in vanilla, as they are just applying a focus to the overflow menu item. Once it is unfocused the menu item will disappear

File: src/dialog/overflow-menu/overflow-menu.directive.ts (hunk 1 above)
Comment (youda97) [reply src/dialog/overflow-menu/overflow-menu.directive.ts]: Well the design is to have the menu item only appear when you hover on it and to only stay visible when the menu is open

**Comment by asudoh** [comment]
Another weigh-in - Would you guys want to open an issue against our vanilla repo if you think the trigger button should be visible while it's open, regardless of the focus state? Meanwhile, do you guys have a way to have your own CSS to override it (instead of using in-line style)?

**Comment by cal-smith** [comment]
For sure ... I think it makes sense to have it explicitly styled ... we should be able to use an inline `styles:[]` property, if not on the directive, then on the wrapper component.

**Pull request merged**